errors/error-handling.js

```diff
@@ -0,0 +1,16 @@
+exports.handlePsqlErrors = (err, req, res, next) => {
+  if (err.code) {
```

Comment:
Nice use of error handling middleware!
These codes are present on PSQL errors, but different codes happen for different PSQL errors
At the moment this works, but in the future you may need to check for specific codes to covers different PSQL errors.